apps/server/prisma/schema.prisma

```diff
@@ -192,3 +174,56 @@ model Subscription {
   isActive  Boolean
   createdAt DateTime @default(now())
 }
+
+model password_reset_tokens {
+  id          Int       @id @default(autoincrement())
+  user_id     Int?
+  reset_token String
+  expires_at  DateTime  @db.Timestamp(6)
+  used        Boolean?  @default(false)
+  created_at  DateTime? @default(now()) @db.Timestamp(6)
+  users       users?    @relation(fields: [user_id], references: [id], onDelete: NoAction, onUpdate: NoAction)
```

Comment:
Updated the relation field names to match the refactored model names (user instead of users), improving consistency.